### PR TITLE
Added tests for Minter harvesting over 5 years

### PR DESCRIPTION
**Pull request by matt-lough**
Updated TokenMinter so that time can be mocked. Added tests for the harvest function

### CODE REVIEW

File: test/lattoken_minter_tests.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const hour = 60 * 60;
+const day = 24 * hour;
+
+const minterStartTime = 1503399600;
+const tokenPerDay = 328767123287;
+
+contract("LATokenMinter", function(accounts) {
+  beforeEach(async () => {
+    token = await LATtoken.new();
+    console.log('new token');
```
Comment (matt-lough) [inline test/lattoken_minter_tests.js]: 😢

**Review by OperationNine — APPROVED** [review]

File: contracts/LATokenMinter.sol [hunk 2]
```diff
@@ -121,7 +121,6 @@ contract LATokenMinter {
         uint wasNotHarvested = maxCurrentHarvest.sub(alreadyHarvestedTokens);
 
         require(token.issueTokens(teamPoolForFrozenTokens, wasNotHarvested));
-
```
Comment (IsaacPayne) [inline contracts/LATokenMinter.sol:124]: ???

File: test/lattoken_minter_tests.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+let LATminter = artifacts.require("LATokenMinterMock");
+let LATtoken = artifacts.require("LATToken");
+const assertFail = require("./helpers/assertFail");
+
+let minter, token;
+const minute = 60
+const hour = 60 * 60;
```
Comment (IsaacPayne) [inline test/lattoken_minter_tests.js]: 60 * minute?

File: test/lattoken_minter_tests.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+const day = 24 * hour;
+
+const minterStartTime = 1503399600;
```
Comment (IsaacPayne) [inline test/lattoken_minter_tests.js]: Maybe a comment on the origin of this number?

**Review by IsaacPayne — APPROVED** [review]

**Pull request merged**